src/views/Orders.vue

```diff
@@ -231,26 +232,22 @@ export default defineComponent({
     },
     async readyForPickup (order: any, shipGroup: any) {
       const pickup = this.getShipmentMethod(shipGroup, order.items) === 'STOREPICKUP';
-      const header = pickup ? this.$t('Ready for pickup') : this.$t('Ready to ship');
-      const message = pickup ? this.$t('An email notification will be sent to that their order is ready for pickup. This order will also be moved to the packed orders tab.', { customerName: order.customerName, space: '<br/><br/>'}) : '';
 
-      const alert = await alertController
-        .create({
-          header: header,
-          message: message,
-          buttons: [{
-            text: this.$t('Cancel'),
-            role: 'cancel'
-          },{
-            text: header,
-            handler: () => {
-              this.store.dispatch('order/quickShipEntireShipGroup', {order, shipGroupSeqId: shipGroup, facilityId: this.currentFacility.facilityId}).then((resp) => {
-                if (resp.data._EVENT_MESSAGE_) this.getPickupOrders();
-              })
-            }
-          }]
+      if(pickup) {
+        const bgJobModal = await  modalController.create({
+          component: AssignPickerModal,
+          componentProps: { order }
         });
-      return alert.present();
+
+        bgJobModal.onDidDismiss().then((data) => {
+          if(data['data']?.dismissed) {
```

Comment:
Improve variable names for data and dismissed

Reply:
Improved Sir